src/compiler/xslt/compile/compile-scenario.xsl

```diff
@@ -127,6 +127,12 @@
                            <xsl:apply-templates select="." mode="x:node-constructor" />
                         </xsl:with-param>
                      </xsl:call-template>
+                     <xsl:if test="self::x:context and not($pending-p) and following-sibling::x:expect">
```

Comment:
I think this `not($pending-p) and following-sibling::x:expect"` is practically the same as `not($pending-p) and x:expect` in the line 151.
If so, it would be nice if we could replace them with a single boolean variable ([`$run-sut-now` in the XQuery compiler](https://github.com/xspec/xspec/blob/3d1b2e1cd6f93fb14c695ff965a5a875e6503500/src/compiler/xquery/compile/compile-scenario.xsl#L113)).

Reply:
Addressed in 7af7ae68ddcbdfe64043c56cdd4c6fbb87a91918